server/lib/rethrown-error.js

```diff
@@ -40,7 +40,7 @@ class RethrownError extends ExtendedError {
     const messageLines = (this.message.match(/\n/g) || []).length + 1;
 
     const indentedOriginalError = error.stack
-      .split('\n')
+      .split(/\r?\n/)
```

Comment:
Fix multiple newlines appearing from child errors we emit from `stderr` output (Windows problem).